Source/Renderer/ComputeCommand.js

```diff
@@ -0,0 +1,102 @@
+/*global define*/
+define([
+        '../Core/defaultValue',
+        '../Core/PrimitiveType',
+        '../Scene/Pass'
+    ], function(
+        defaultValue,
+        PrimitiveType,
+        Pass) {
+    "use strict";
+
+    /**
+     * Represents a command to the renderer for GPU Compute (using old-school GPGPU).
+     *
+     * @private
+     */
+    var ComputeCommand = function(options) {
+        options = defaultValue(options, defaultValue.EMPTY_OBJECT);
+
+        /**
+         * The vertex array. If none is provided, a viewport quad will be used.
+         *
+         * @type {VertexArray}
+         * @default undefined
+         */
+        this.vertexArray = options.vertexArray;
+
+        /**
+         * The fragment shader source. The default vertex shader is ViewportQuadVS.
+         *
+         * @type {ShaderSource}
+         * @default undefined
+         */
+        this.fragmentShaderSource = options.fragmentShaderSource;
+
+        /**
+         * The shader program to apply.
+         *
+         * @type {ShaderProgram}
+         * @default undefined
+         */
+        this.shaderProgram = options.shaderProgram;
+
+        /**
+         * An object with functions whose names match the uniforms in the shader program
+         * and return values to set those uniforms.
+         *
+         * @type {Object}
+         * @default undefined
+         */
+        this.uniformMap = options.uniformMap;
+
+        /**
+         * Texture to use for offscreen rendering.
+         *
+         * @type {Texture}
+         * @default undefined
+         */
+        this.outputTexture = options.outputTexture;
+
+        /**
+         * Whether this command will persist beyond this call.
```

Comment:
It's not the command that would persist; it is the renderer resources, i.e., framebuffer, etc.